src/stackanalysismodule.ts

```diff
@@ -46,12 +46,11 @@ export module stackanalysismodule {
               let thatContext: any;
               options['uri'] = `${
                 Apiendpoint.STACK_API_URL
-              }stack-analyses/?user_key=${Apiendpoint.STACK_API_USER_KEY}`;
+                }stack-analyses/?user_key=${Apiendpoint.STACK_API_USER_KEY}`;
               options['formData'] = payloadData;
               options['headers'] = {
                 showTransitiveReport: 'true',
-                origin: 'vscode',
-                ecosystem: Apiendpoint.API_ECOSYSTEM
+                'x-3scale-account-secret': 'not-set'
```

Comment:
This must be a local change, don't add it for review.

This is no longer needed if you are working with staging